nodejs/package.json

```diff
@@ -18,6 +18,7 @@
     "test:jsonrpc:mainnet:contract:deploy": "jest json-rpc-relay/integration.tests/contracts/mainnet/deploy.contract.mainnet.test.js",
     "test:jsonrpc:mainnet:contract:call": "jest json-rpc-relay/integration.tests/contracts/mainnet/call.contract.mainnet.test.js",
     "test:jsonrpc:mainnet:contract:write": "jest json-rpc-relay/integration.tests/contracts/mainnet/call.contract.mainnet.test.js",
+    "test:jsonrpc:polygon": "jest json-rpc-relay/polygon/*",
```

Comment:
Nice ,perhaps we can change it to `test:jsonrpc:polygon:read` and do writing a bit later